tests/kernel/blktests.pm

```diff
@@ -17,6 +17,26 @@ use testapi;
 use utils;
 use repo_tools 'add_qa_head_repo';
 
+#can accept tags: none, all, specific
+sub _test_devices {
+    my ($devices) = @_;
+
+    if ($devices eq 'all') {
+        $devices = script_output('lsblk -d -n -p -o NAME');
+        $devices = (join ' ', split ' ', $devices);
+    } elsif ($devices eq 'specific') {
+        die('BLK_DEVICE_ONLY: specific - not implemented');
+    } elsif ($devices eq 'none') {
```

Comment:
Wouldn't be better anything else besides all/specific treated as none? We could spare `if/else` and `die` lines.

Reply:
ah sure, good point. will change it.

Reply:
I simply do only 'none' or else now. I will mark this resolve for now and we can talk about v2 soonish.